content/blog/posts/2022-08-04-add-native-notifications.md

```diff
@@ -0,0 +1,172 @@
+---
+authorId: chuckcarpenter
+categories:
+  - notifications
+  - javascript
+date: ''
+slug: add-desktop-notifications
```

Comment:
`slug` and the end of the file name need to match. So if the file name is correct, this should be `add-native-notifications`. Do these work on phones too or just desktop?